hoomd/ForceCompute.h

```diff
@@ -75,6 +75,11 @@ class PYBIND11_EXPORT ForceCompute : public Compute
     //! Computes the forces
     virtual void compute(uint64_t timestep);
 
+    virtual bool checkSurrounding()
+        {
+        return false;
+        }
+
```

Comment:
I think this method can be moved to `MeshForceCompute` as well. It is only used by `MeshDynamicBondUpdater` which operates on a list of `MeshForceCompute` objects.